### PR TITLE
#524 login button   fix linebreak

### PR DESCRIPTION
**Pull request by Kobraz**
Done these changes in src/scss/custom/_headers.css:

#navUI --> Added white-space: nowrap;

image

### CODE REVIEW

**Comment by Kobraz** [comment]
The first thing I did will only affect the page smaller thatn 980px, and the line-break happen around 1250px, so I added the class "login-button" to the login button in pages/auth/login/index.html and in _header.scss I added

.login-button {
white-space: nowrap;
}

This will make sure that the login button text never break into to text-lines.

image

File: pages/TBD/listings.html
Comment (Olekrr) [inline pages/TBD/listings.html:29]: Paths to images should not be changed untill the vite.config has been tried, goes for all path changes in this PR

File: src/scss/custom/_header.scss
Comment (Olekrr) [inline src/scss/custom/_header.scss:14]: Looks good to me

**Review by Olekrr — CHANGES_REQUESTED** [review]
Button looks good to me.

However this PR contains path changes for images and icons. a different group is currently working on the vite configuration, which should solve the way paths from the public folder is being served in the production build.

These path changes needs to be removed before I can approve the PR.

File: pages/TBD/listings.html
Comment (Kobraz) [reply pages/TBD/listings.html:29]: But that was the last ticket - not this one... I changed the link til search.svg in #454 Search Icons using wrong src

I can  understand if that one was not done correct, but how are that ticket related to this PR?

Comment (Kobraz) [reply pages/TBD/listings.html:29]: Anyway - I understand someone else are working with this atm?

Comment (Olekrr) [reply pages/TBD/listings.html:29]: The ticket #454 has been put on hold, this is correct. However the changes you did there, that we did not merge is also part of this pull request. It is in the commit history, and you can see it here under files changed. 

This PR is for #524 and should not contain the earlier fixes you did for #454

File: pages/admin/index.html
Comment (Olekrr) [inline pages/admin/index.html:75]: here

File: pages/admin/index.html
Comment (Olekrr) [inline pages/admin/index.html:104]: here

File: pages/admin/index.html
Comment (Olekrr) [inline pages/admin/index.html:156]: here

**Comment by Kobraz** [comment]
Removed changes from ticket #454

File: src/scss/custom/_header.scss
Comment (Kobraz) [reply src/scss/custom/_header.scss:14]: Good :)

**Review by Olekrr — APPROVED** [review]

**Comment by Olekrr** [comment]
Great, thank you for implementing the requested changes - merging this now

**Pull request merged**
